sdk/src/main/java/com/qonversion/android/sdk/internal/userProperties/PendingUserPropertiesStorage.kt

```diff
@@ -0,0 +1,12 @@
+package com.qonversion.android.sdk.internal.userProperties
+
+internal interface PendingUserPropertiesStorage {
+
+    fun getProperties(): Map<String, String>
+
+    fun save(key: String, value: String)
+
+    fun delete(key: String)
+
+    fun update(key: String, newValue: String)
```

Comment:
Do we need this method? I'd advise combining both `save` and `update` methods into one `set`